src/main/java/org/smof/annnotations/SmofPFEQuery.java

```diff
@@ -0,0 +1,17 @@
+package org.smof.annnotations;
```

Comment:
This annotation is supposed to be a wrapper around `@SmofQueryA` right? If so, why is it needed?